feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -104,6 +114,21 @@ fun TimetableGridItem(
         it.copy(fontSize = titleFontSize, lineHeight = titleLineHeight, color = textColor)
     }
 
+    var displayedPosition by remember { mutableFloatStateOf(0f) }
+    var itemHeight by remember { mutableIntStateOf(0) }
+    var titleHeight by remember { mutableIntStateOf(0) }
+    var timeHeight by remember { mutableIntStateOf(0) }
+    var speakerHeight by remember { mutableIntStateOf(0) }
```

Comment:
theere are possibility that height of column composable for title, time speaker have a variety of values according to session name.
so here, getting each height and calculating `maxOffset` from these values.